alacritty_terminal/src/config/mod.rs

```diff
@@ -133,6 +133,10 @@ pub struct Config<T> {
     #[serde(flatten)]
     pub ui_config: T,
 
+    /// Remain open after child process exits
+    #[serde(default, deserialize_with = "failure_default")]
```

Comment:
FYI, you can still set this option in `alacritty.yml`. To fix this issue you should replace current `#[serde(...)` with `#[serde(skip)`.

Reply:
This is a good idea.